website/server/controllers/api-v3/quests.js

```diff
@@ -372,12 +372,12 @@ api.cancelQuest = {
 
     let questName = questScrolls[group.quest.key].text('en');
     const newChatMessage = group.sendChat(`\`${user.profile.name} cancelled the party quest ${questName}.\``);
-    await newChatMessage.save();
 
     group.quest = Group.cleanGroupQuest();
     group.markModified('quest');
 
     let [savedGroup] = await Promise.all([
+      newChatMessage.save(),
```

Comment:
thanks! last thing, putting newChatMessage.save() at the beginning of the array will cause savedGroup to get its value instead of group.save(). It should be at least in second place inside the array

Reply:
Sorry that I couldn't do it from the first commit(

Thanks for the review!